README.md

```diff
@@ -18,138 +18,9 @@ The [complete documentation](https://python-poetry.org/docs/) is available on th
-
-```bash
-curl -sSL https://install.python-poetry.org | POETRY_HOME=/etc/poetry python3 -
-```
-
-If you want to install prerelease versions, you can do so by passing `--preview` option to `install-poetry.py`
-or by using the `POETRY_PREVIEW` environment variable:
-
-```bash
-curl -sSL https://install.python-poetry.org | python3 - --preview
-curl -sSL https://install.python-poetry.org | POETRY_PREVIEW=1 python3 -
-```
-
-Similarly, if you want to install a specific version, you can use `--version` option or the `POETRY_VERSION`
-environment variable:
-
-```bash
-curl -sSL https://install.python-poetry.org | python3 - --version 1.2.0
-curl -sSL https://install.python-poetry.org | POETRY_VERSION=1.2.0 python3 -
-```
-
-You can also install Poetry for a `git` repository by using the `--git` option:
-
-```bash
-curl -sSL https://install.python-poetry.org | python3 - --git https://github.com/python-poetry/poetry.git@master
-````
-
-_Note that the installer does not support Python < 3.7._
-
-## Updating `poetry`
-
-Updating poetry to the latest stable version is as simple as calling the `self update` command.
-
-**Warning**: Poetry versions installed using the now deprecated `get-poetry.py` installer will not be able to use this
-command to update to 1.2 releases or later. Migrate to using the `install-poetry.py` installer or `pipx`.
-
-```bash
-poetry self update
-```
-
-If you want to install prerelease versions, you can use the `--preview` option.
-
-```bash
-poetry self update --preview
-```
-
-And finally, if you want to install a specific version you can pass it as an argument
-to `self update`.
-
-```bash
-poetry self update 1.2.0
-```
-
-
-## Enable tab completion for Bash, Fish, or Zsh
-
-`poetry` supports generating completion scripts for Bash, Fish, and Zsh.
-See `poetry help completions` for full details, but the gist is as simple as using one of the following:
-
-```bash
-# Bash
-poetry completions bash > /etc/bash_completion.d/poetry.bash-completion
-
-# Fish
-poetry completions fish > ~/.config/fish/completions/poetry.fish
-
-# Zsh
-poetry completions zsh > ~/.zfunc/_poetry
-
-# Zsh (Oh-My-Zsh)
-mkdir $ZSH_CUSTOM/plugins/poetry
-poetry completions zsh > $ZSH_CUSTOM/plugins/poetry/_poetry
-rm ~/.zcompdump*
-# add `poetry` in the `plugins` list (https://github.com/ohmyzsh/ohmyzsh#enabling-plugins)
-
-# Zsh (prezto)
-poetry completions zsh > ~/.zprezto/modules/completion/external/src/_poetry
-```
-
-*Note:* you may need to restart your shell in order for the changes to take
-effect.
-
-For `zsh`, you must then add the following line in your `~/.zshrc` before
-`compinit` (not for homebrew setup):
-
-```zsh
-fpath+=~/.zfunc
-```
-
+Instructions on how to install `poetry` can be found [here](https://python-poetry.org/docs/master/#installation).
+You can also refer [here](https://python-poetry.org/docs/master/#enable-tab-completion-for-bash-fish-or-zsh) for
+information on how to enable tab completion in your environment.
```

Comment:
A lot of people seems to find it confusing, that the instructions in the README of the master branch doesn't reflect the latest stable version. Maybe we can make it more clear by linking to both (master, latest stable) documentations?

Reply:
Partially the reason why this PR was created. I was deliberitely avoiding having both listed for now as we are transitioning installation methods. Specifically, we would want avoid folks going to stable one by default. With this change the README will no longer explicitly contain this instruction anyway. And further, going forward we really want to avoid having version specific installation methods. With that in mind, having both listed here is less likely to help the issue.